.cargo/config.toml

```diff
@@ -1,4 +1,4 @@
 [target.'cfg(target_os = "linux")']
 rustflags = [
-  "-C", "link-arg=-Wl,-E"
+  "-C", "link-arg=-Wl,-E",
```

Comment:
(No action required.) Could you explain?

Reply:
Something I forgot to remove while testing different configurations but I like having trailing-comma when it's allowed to reduce diff when adding lines into lists.